pontoon/api/urls.py

```diff
@@ -14,6 +14,61 @@
         r"^graphql/?$",
         csrf_exempt(GraphQLView.as_view(schema=schema, graphiql=True)),
     ),
+    path(
+        "api/new/",
+        include(
+            [
+                path(
+                    # Locales
+                    "locales/",
+                    views.LocaleListView.as_view(),
+                    name="locale-list",
+                ),
+                path(
+                    # Locale
+                    "locales/<str:code>",
+                    views.LocaleIndividualView.as_view(),
+                    name="locale-individual",
+                ),
+                path(
+                    # Projects
+                    "projects/",
+                    views.ProjectListView.as_view(),
+                    name="project-list",
+                ),
+                path(
+                    # Project
+                    "projects/<str:slug>",
+                    views.ProjectIndividualView.as_view(),
+                    name="project-individual",
+                ),
+                path(
+                    # Localizations
+                    "localizations/",
+                    views.ProjectLocaleListView.as_view(),
+                    name="localization-list",
+                ),
```

Comment:
Do we need this? This is listing all project-locale combinations in one go, yes? That seems like a lot.

Reply:
I'd scratch that.